### ISSUE TITLE
YOLOv4: Optimal Speed and Accuracy of Object Detection

### ISSUE THREAD

**Issue by NoahHalonen** [NoahHalonen/NoahHalonen-Capstone_Resources#21]
### Title
YOLOv4: Optimal Speed and Accuracy of Object Detection

### URL
https://arxiv.org/abs/2004.10934

### Summary 
YOLOv4 presents a state-of-the-art object detection model optimized for speed and accuracy, capable of real-time performance on a single GPU. Incorporating new features such as Weighted-Residual-Connections (WRC), Cross-Stage-Partial connections (CSP), Self-adversarial-training (SAT), Mish-activation, and others, YOLOv4 achieves top results on the MS COCO dataset with 43.5% AP at approximately 65 FPS on a Tesla V100 GPU. This paper details the contributions towards developing a highly efficient and powerful object detection model accessible for training on conventional GPUs like the 1080 Ti or 2080 Ti, alongside an exploration of the impact of various state-of-the-art enhancement methods on the training process.

### Key Points 
- YOLOv4 introduces several new features to improve Convolutional Neural Network (CNN) accuracy without significantly impacting inference speed, including CSP, CmBN, SAT, Mish activation, Mosaic data augmentation, DropBlock regularization, and CIoU loss.
- The model aims for real-time operation on conventional GPUs, facilitating mass usage at an affordable price and enabling the use of highly accurate object detectors in a variety of applications beyond recommendation systems, such as real-time process management and reduction of human input.
- YOLOv4's architecture comprises the CSPDarknet53 backbone, SPP additional module, PANet path-aggregation neck, and the YOLOv3 head, optimized for both speed and accuracy through extensive experimentation and tuning.
- The paper elaborates on the methodology behind selecting YOLOv4's architecture, including the balance between input network resolution, the number of convolutional layers, and the parameter number, emphasizing the necessity of a large receptive field and high model capacity for detecting multiple objects.
- It presents a thorough analysis of the impact of various "Bag of Freebies" and "Bag of Specials" on the performance of object detection training, contributing significantly to the advancement of efficient training and high-performance object detection models.

### Citation
@misc{bochkovskiy2020yolov4,
      title={YOLOv4: Optimal Speed and Accuracy of Object Detection}, 
      author={Alexey Bochkovskiy and Chien-Yao Wang and Hong-Yuan Mark Liao},
      year={2020},
      eprint={2004.10934},
      archivePrefix={arXiv},
      primaryClass={cs.CV}
}
